src/test/java/com/puppycrawl/tools/checkstyle/xpath/ElementNodeTest.java

```diff
@@ -58,6 +59,61 @@ public void init() throws Exception {
         rootNode = new RootNode(rootAst);
     }
 
+    @Test
+    public void testNodeOrdering() throws Exception {
```

Comment:
I am not sure `ElementNodeTest` is appropriate class for this method, please move it to `XpathMapperTest`